src/vitabel/vitals.py

```diff
@@ -1156,30 +1158,41 @@ def shocks(self):
 
     def compute_etco2_and_ventilations(
         self,
-        mode="threshold",
-        breaththresh: float = 4,
-        etco2_thresh: float = 4,
+        mode: Literal['threshold', 'filter'] = 'filter',
+        breath_thresh: float = 2,
+        etco2_thresh: float = 3,
+        **kwargs,
     ):
-        """Computes the etCO2 values and the timestamps of the ventilations based
-        on CO2 waveform and add it to the labels.
+        """
+        Computes end-tidal CO2 (etCO₂) values and timestamps of ventilations from the capnography waveform,
+        and adds them as labels.
 
-        The capnography signal must be a channel of the file and needs to be named 'capnography'. Currently two different versions are implemented: 'threshold' and 'filter'. The default is 'filter'.
+        The capnography signal must be present as a channel named 'capnography'. Two detection methods are supported:
+        - 'filter': An unpublished method by Wolfgang Kern (default).
+        - 'threshold': The method described by Aramendi et al., "Feasibility of the capnogram to monitor ventilation rate during cardiopulmonary resuscitation"
+          (Resuscitation, 2016, DOI: `10.1016/j.resuscitation.2016.08.033 <https://doi.org/10.1016/j.resuscitation.2016.08.033>`_).
 
         Parameters
         ----------
-        mode : str, optional,
-            Which method to use to detect ventilations from CO2 signal. Either 'filter' which is a unpublished method by Wolfgang Kern or 'threshold',
-            which is the method presented by Aramendi et al. "Feasibility of the capnogram to monitor ventilation rate during cardiopulmonary resuscitation" `10.1016/j.resuscitation.2016.08.033 <https://doi.org/10.1016/j.resuscitation.2016.08.033>`_
-        breaththresh : float, optional
-            Threshold value below which a minimum is identified as ventilation/respiration . The default is 2 (mmHg).
+        mode : {'filter', 'threshold'}, optional
```

Comment:
Type annotations come from the actual type hints, no need to specify them again

```suggestion
        mode
```